modules/core/src/test/java/org/apache/ignite/internal/processors/cache/query/GridCacheQueryTransformerSelfTest.java

```diff
@@ -60,15 +59,7 @@
     /** {@inheritDoc} */
     @Override protected void beforeTestsStarted() throws Exception {
         startGridsMultiThreaded(3);
-
-        try {
-            Ignition.setClientMode(true);
-
-            startGrid();
-        }
-        finally {
-            Ignition.setClientMode(false);
-        }
+        startGrid();
```

Comment:
To start client grid `startClientGrid()` should be called.

Reply:
Fixed.